internal/renderers/skia/lib.rs

```diff
@@ -49,13 +49,16 @@ cfg_if::cfg_if! {
 
 /// Use the SkiaRenderer when implementing a custom Slint platform where you deliver events to
 /// Slint and want the scene to be rendered using Skia as underlying graphics library.
-pub struct SkiaRenderer {
+pub struct SkiaRenderer<NativeWindowWrapper> {
     window_adapter_weak: Weak<dyn WindowAdapter>,
     rendering_notifier: RefCell<Option<Box<dyn RenderingNotifier>>>,
-    canvas: RefCell<Option<SkiaCanvas<DefaultSurface>>>,
+    canvas: RefCell<Option<SkiaCanvas<DefaultSurface, NativeWindowWrapper>>>,
 }
 
-impl SkiaRenderer {
+impl<
+        NativeWindowWrapper: raw_window_handle::HasRawWindowHandle + raw_window_handle::HasRawDisplayHandle,
+    > SkiaRenderer<NativeWindowWrapper>
+{
     /// Creates a new renderer is associated with the provided window adapter.
     pub fn new(window_adapter_weak: &Weak<dyn WindowAdapter>) -> Self {
```

Comment:
Btw, this is not the same API as the SoftwareRenderer,  which  takes the window_adapter_weak by value.

Reply:
Ah, I'll change that. Well spotted, thanks.